Neos.Flow/Classes/Reflection/ReflectionService.php

```diff
@@ -2063,17 +2049,14 @@ public function freezePackageReflection($packageKey)
      * Filter an array of entries were keys are class names by being in the given package namespace.
      *
      * @param array $array
-     * @param string $packageNamespace
+     * @param string $packageKey
      * @return array
      */
-    protected function filterArrayByClassesInNamespace(array $array, $packageNamespace)
+    protected function filterArrayByClassesInNamespace(array $array, $packageKey)
```

Comment:
Guess this method could be slightly renamed now (`filterArrayByClassesInPackage` or sth), but not sure about the consequences... so let it be.